mmdet/models/anchor_heads/fcos_head.py

```diff
@@ -0,0 +1,257 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import normal_init
+
+from mmdet.core import (centerness_target, fcos_target, sigmoid_focal_loss,
```

Comment:
Implement `centerness_target` and `fcos_target` as methods of `FCOSHead`. There will be a refactoring for `anchor_target`.